<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Films</class>
 <widget class="QDialog" name="Films">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="2" colspan="2">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageSelect">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTableView" name="tableViewSelect"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReloadTbSelect">
         <property name="text">
          <string>Перезагрузить таблицу</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageAdd">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditTitle"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditGenres"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="lineEditDirector"/>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>* - important fields</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelDirector">
           <property name="text">
            <string>Режисссер</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QVBoxLayout" name="verticalLayoutReceipDate">
           <item>
            <widget class="QLabel" name="labelReceiptDate">
             <property name="text">
              <string>Дата получения*</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="2" column="0">
              <widget class="QLineEdit" name="lineEditDay"/>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEditMonth"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelDay">
               <property name="text">
                <string>день</string>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QLabel" name="labelMinutes">
               <property name="text">
                <string>минуты</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEditYear"/>
             </item>
             <item row="1" column="3">
              <widget class="QLabel" name="labelHours">
               <property name="text">
                <string>часы</string>
               </property>
              </widget>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="lineEditHours"/>
             </item>
             <item row="2" column="4">
              <widget class="QLineEdit" name="lineEditMinutes"/>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="labelMonth">
               <property name="text">
                <string>месяц</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="labelYear">
               <property name="text">
                <string>год</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="labelGenres">
           <property name="text">
            <string>Жанр(ы)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelTitle">
           <property name="text">
            <string>Название*</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0" colspan="2">
          <widget class="QPushButton" name="pushButtonAddFilm">
           <property name="text">
            <string>Добавить фильм</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageDelete">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonReloadTbDelete">
         <property name="text">
          <string>Перезагрузить таблицу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonDeleteFilm">
         <property name="text">
          <string>Удалить выбранный фильм</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Закрыть окно</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
